widgets/HTTP-batchsink.json

```diff
@@ -151,6 +130,81 @@
             "default": "60000"
           }
         },
+        {
+          "widget-type": "keyvalue-dropdown",
+          "label": "HTTP Errors Handling",
+          "name": "httpErrorsHandling",
+          "widget-attributes": {
+            "default": "2..:Success,.*:Fail",
+            "showDelimiter": "false",
+            "dropdownOptions": [
+              "Success",
+              "Fail",
+              "Skip",
+              "Send to error",
+              "Retry and fail",
+              "Retry and skip",
+              "Retry and send to error"
+            ],
+            "key-placeholder": "HTTP Status Code Regex"
+          }
+        },
+        {
+          "widget-type": "radio-group",
+          "label": "Non-HTTP Error Handling",
+          "name": "errorHandling",
+          "widget-attributes": {
+            "layout": "inline",
+            "default": "stopOnError",
+            "options": [
+              {
+                "id": "stopOnError",
+                "label": "Stop on error"
+              },
+              {
+                "id": "skipOnError",
+                "label": "Skip on error"
+              }
+            ]
+          }
+        },
+        {
+          "widget-type": "radio-group",
+          "label": "Retry Policy",
+          "name": "retryPolicy",
+          "widget-attributes": {
+            "layout": "inline",
+            "default": "exponential",
+            "options": [
+              {
+                "id": "exponential",
+                "label": "Exponential"
+              },
+              {
+                "id": "linear",
+                "label": "Linear"
+              }
+            ]
+          }
+        },
+        {
+          "widget-type": "number",
+          "label": "Linear Retry Interval",
+          "name": "linearRetryInterval",
+          "widget-attributes": {
+            "min": "0",
+            "default": "30"
+          }
```

Comment:
I suggest we add a filter to only show this field when `Retry Policy` is Linear.

Reply:
Added filter to show the Linear Retry interval only if the policy selected is Linear. 4e4ee6211be03c9e368a811866c8d96ec49518d3